tailwind.config.js

```diff
@@ -47,6 +23,7 @@ module.exports = {
     extend: {
       minHeight: {
         '28': '7rem'
+        // What are you trying to do here? This doesn't make sense with the documentation: https://tailwindcss.com/docs/min-height#customizing.
```

Comment:
https://github.com/MikeRogers0/Ruby-Meetup-Calendar/blob/61a95606e71026c9cc03b6d99a43da79a7378c6c/frontend/styles/components/calendar.css#L54-L56

This gives us the class `.min-h-28`